packages/terra-core-docs/src/terra-dev-site/doc/show-hide/About.1.doc.mdx

```diff
@@ -42,9 +46,113 @@ The Show-Hide component must be composed inside the [Base][1] component with a l
 ## Usage
 
 ```jsx
-import ShowHide from 'terra-show-hide';
+import ShowHide, { Focuser } from 'terra-show-hide';
 ```
 
+## Accessibility
+
+<Notice variant="important" ariaLevel="3">
+
+  To make the Show/Hide component accessible, you must use the `focusRef` prop. This prop allows assistive technologies such as screen readers to put focus on the hidden text and announce it when it is disclosed. The `focusRef` is optional for backward compatibility but is required to comply with accessibility requirements.
+
+</Notice>
+
+#### Truncate a paragraph of text
```

Comment:
Even though the Notice above gives an h3, this feels like these headings should be level 3 as well since the header in the notice should only apply to what's grouped within it.

Reply:
Thank you! Changed the headings level to be level 3: https://github.com/cerner/terra-core/pull/3838/commits/09d4e4543413b7aa554d30b921cfe7f1ec1455b4